lib/utils.js

```diff
@@ -253,6 +253,15 @@ const utils = {
       .digest('hex');
   },
 
+  getShallowLocalStatus: async (dir, globPattern) => {
+    const dirPattern = path.resolve(path.join(dir, globPattern));
+    const files = glob.sync(dirPattern);
+    const relFiles = await Promise.all(files.map(async (file) => {
```

Comment:
you could just return this promise.

Reply:
you can also use mapP from omnibelt, and you would get rid of the Promise.all. 

Also you dont have to do these suggestions more so that you are aware of your options :)